package.json

```diff
@@ -65,5 +65,8 @@
     "route",
     "routes",
     "router"
-  ]
+  ],
+  "system": { 
+    "main": "lib/umd/ReactRouter.min" 
```

Comment:
Are you sure you want to use the minified file, @omerts? Seems like bundlers would want the unminified file.